### PR TITLE
FOGL-6566 Update architecture documentation

### PR DESCRIPTION
**Pull request by MarkRiddoch**
Signed-off-by: Mark Riddoch <mark@dianomic.com>

### CODE REVIEW

File: docs/fledge_architecture.rst [hunk 1]
```diff
@@ -62,15 +63,15 @@ Core functionality includes:
...
 
-Southbound Microservices
-========================
+South Microservices
+===================
 
-Southbound microservices offer bi-directional communication of data and metadata between Edge devices, such as sensors, actuators or PLCs and Fledge. Smaller systems may have this service installed onboard Edge devices. Southbound components are typically deployed as always-running services, which continuously wait for new data. Alternatively, they can be deployed as single-shot tasks, which periodically spin up, collect data and spin down.
+South microservices offer bi-directional communication of data and metadata between Edge devices, such as sensors, actuators or PLCs and Fledge. Smaller systems may have this service installed onboard Edge devices. Southbound components are typically deployed as always-running services, which continuously wait for new data. Alternatively, they can be deployed as single-shot tasks, which periodically spin up, collect data and spin down.
```
Comment (ashish-jabble) [inline docs/fledge_architecture.rst:71]: >Southbound components

South components?

File: docs/fledge_architecture.rst [hunk 2]
```diff
@@ -95,16 +96,31 @@ A sample of existing Filters:
 
 **Rate**: buffer data but don’t send it, then if an error condition occurs, send the previous data.
 
-Event Engine
-============
+**Contrast**: Enhance the contrast of image type data
```
Comment (ashish-jabble) [inline docs/fledge_architecture.rst:99]: This filter plugin is not available at Fledge side.

**Review by ashish-jabble — DISMISSED** [review]
fyi, there is autocorrection text underlined for CoAP in fledge_architecture.png.

File: docs/fledge_architecture.rst (hunk 2 above)
Comment (MarkRiddoch) [reply docs/fledge_architecture.rst:99]: Not too worried about this

File: docs/fledge_architecture.rst [hunk 3]
```diff
@@ -62,15 +63,15 @@ Core functionality includes:
...
+South microservices offer bi-directional communication of data and metadata between Edge devices, such as sensors, actuators or PLCs and Fledge. Smaller systems may have this service installed onboard Edge devices. South components are typically deployed as always-running services, which continuously wait for new data. Alternatively, they can be deployed as single-shot tasks, which periodically spin up, collect data and spin down.
 
-Northbound Microservices
-========================
+North Microservices
+===================
 
 Northbound microservices offer bi-directional communication of data and metadata between the Fledge platform and larger systems located locally or in the cloud. Larger systems may be private and public Cloud data services, proprietary solutions or Fledge instances with larger footprints. Northbound components are typically deployed as one-shot tasks, which periodically spin up and send data which has been batched, then spin down. However, they can also be deployed as continually-running services.
```
Comment (ashish-jabble) [inline docs/fledge_architecture.rst:76]: Northbound-> North

**Pull request merged**